src/hazelcore/Action_Exp.re

```diff
@@ -1511,12 +1517,16 @@ and syn_perform_operand =
   /* \x :<| Int . x + 1   ==>   \x| . x + 1 */
   | (Backspace, CursorE(OnDelim(1, After), Lam(_, p, body))) =>
     let (p, body_ctx, u_gen) =
-      Statics_Pat.ana_fix_holes(ctx, u_gen, p, Hole);
+      Statics_Pat.ana_fix_holes(ctx, u_gen, p, Unknown(Internal));
```

Comment:
I think unrelated but maybe could cause related problems -- this seems incorrect? we should be synthesizing the argument pattern for a lambda in synthetic position right?

Reply:
not really sure but i'll make that change